admin/css/login.css

```diff
@@ -52,26 +35,19 @@ body.login div#login form#loginform {
 }
 
 form#loginform p.galogin {
-  background: none repeat scroll 0 0 #09c;
-  border-color: #0074A2;
-  box-shadow: 0 1px 0 rgba(120, 200, 230, 0.5) inset, 0 1px 0 rgba(0, 0, 0, 0.15);
-  color: #FFFFFF;
-  text-decoration: none;
-  text-align: center;
-  vertical-align: middle;
-  border-radius: 0px;
   padding: 0;
-  height: auto;
-  font-size: 14px;
-  margin-bottom: 6px;
+}
+
+form#loginform p.galogin a {
+  line-height: initial;
```

Comment:
Hey @comzeradd this seems to cause the text to be not centered vertically (at least in Chrome), I guess you could make it `27px` which is the exact height of the button. I would also add `text-decoration: none` to remove the underline.

Reply:
Ignore this, my plugin was outdated..